zerver/lib/avatar.py

```diff
@@ -36,6 +34,33 @@ def avatar_url(
     )
 
 
+def get_system_bots_avatar_file_name(email: str) -> str:
+    # System bot avatar files are stored in the `images/static_avatars/`
```

Comment:
we previously used dict for this logic
```
SYSTEM_BOTS_AVATAR_FILES = {
    # This is also used in zerver/lib/storage.py to ensure
    # these files are hashed when served as static files.
    settings.WELCOME_BOT: "images/welcome-bot.png",
    settings.NOTIFICATION_BOT: "images/logo/zulip-icon-square.svg",
    settings.EMAIL_GATEWAY_BOT: "images/email-gateway-bot.png",
}
```
I thought it would be a bit cleaner if we just matched the email username name, I wonder if it's better to just stick to dict for simplicity..